docs/scheduler-install.md

```diff
@@ -25,7 +25,40 @@ mvn install -Dmaven.test.skip=true
 
 ## Configure
 
-Edit the [Scheduler configuration file](/config/scheduler.conf) file to allow its comunication with the SAPS Catalog and Arrebol Service. This configuration file also customizes the behaviour of the Scheduler, including the frequency that the Scheduler tries to select new task to be submitted.
+Edit the files:
+- [Scheduler configuration file](/config/scheduler.conf) to allow its comunication with the SAPS Catalog and Arrebol Service. This configuration file also customizes the behaviour of the Scheduler, including the frequency that the Scheduler tries to select new task to be submitted.
+- [SAPS Scripts](/resources/execution_script_tags.json) to allow mapping the versions of the scripts with their respective Docker information (tag and repository) to be used in processing. For example:
+    
```

Comment:
didn't we decide to remove below json and just link to the one within the resources directory?

Reply:
The link to json in the resources directory was associated with the term "SAPS Scripts", do you want me to change something? how to remove this json?